NetKAN.schema

```diff
@@ -84,11 +84,11 @@
             "type": "string",
             "oneOf": [
                 {
-                    "pattern": "^#/ckan/ksp-avc(/.*)?",
+                    "pattern": "^#\/ckan\/ksp-avc(\/.*)$",
                     "description": "Version information should be retrieved from an embedded KSP-AVC .version file inside the downloaded file"
                 },
                 {
-                    "pattern": "^#/ckan/space-warp(/.*)?",
+                    "pattern": "^#\/ckan\/space-warp(\/.*)$",
```

Comment:
```suggestion
                    "pattern": "^#/ckan/space-warp(/.*)?$",
```

This capture group is optional, not required. This should be treated as valid:

```json
    "$vref": "#/ckan/space-warp",
```

Reply:
Same as above, `$` fixes what I thought removing `?` would fix, lgtm.